### DISCUSSION TITLE
Running this script on a server behind the proxy

### DISCUSSION THREAD

**Discussion by ritchie-gu** [CrowdStrike/Cloud-AWS#127] [Q&A]
https://github.com/CrowdStrike/Cloud-AWS/blob/c705413f7f60f5ec46d3cf9d656dc0deea6051e6/Agent-Install-Examples/powershell/sensor_install.ps1#L50

Is there an easy way to run it on a server behind an outbound proxy?

**Comment by redhatrises** [comment]
Probably should run the following first:

```netsh winhttp import proxy source=ie ```

which should import the IE proxy configuration into the powershell session

**Comment by bk-cs** [comment]
Similar to what @redhatrises mentioned, here's an article that explains how to set the proxy a slightly different way: http://woshub.com/using-powershell-behind-a-proxy/
